app/services/billing_entities/update_invoice_numbering_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module BillingEntities
+  class UpdateInvoiceNumberingService < BaseService
```

Comment:
I'm thinking: since it doesn't save, should it be "ChangeInvoiceNumberingService", or something like this to have less confusion